content/docs/concepts/authorization/v2.x/release/_index.md

```diff
@@ -7,12 +7,17 @@ Description: >
   Container Storage Modules (CSM) release notes for authorization
 ---
 
-
 ## CSM Authorization 2.1.0
```

Comment:
Should this include the "v" before the version, similar to other release note pages?

Reply:
Once we run the action, it will properly update the version from whatever format is used in https://github.com/dell/csm/blob/main/config/csm-versions.yaml.

There are only changes in the `_index.md` for cleanup purposes.